website/snippets/_config-description-resource.md

```diff
@@ -0,0 +1,3 @@
+Resource-specific configurations are applicable to only one dbt resource type rather than multiple resource types. You can define these settings in the project file (`dbt_project.yml`), a property file (`models/properties.yml` for models, similarly for other resources), or within the resource’s file using the `{{ config() }}` macro.<br />
+
+<span>The following resource-specific config code block is only available to {props.meta.resource_type}</span>:
```

Comment:
I'm going approve in a moment, which will trigger the auto-merge.

But adding this to a follow-up PR might read a little easier:

```suggestion
<span>The following resource-specific configurations are only available to {props.meta.resource_type}</span>:
```

Here's how it looks like right now:

<img width="500" alt="image" src="https://github.com/dbt-labs/docs.getdbt.com/assets/44704949/8ee59a57-d92d-4c37-825e-969f30827b24">